src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Circle extends FiguresState implements FiguresArea, DrawProperties {
+    private final double radius = FigureSupplier.getRandomProperties();
```

Comment:
Do not set random field values directly, pass them through the constructor

Reply:
done